shell/platform/linux/fl_binary_messenger.cc

```diff
@@ -139,18 +142,20 @@ G_MODULE_EXPORT void fl_binary_messenger_set_message_handler_on_channel(
 
 G_MODULE_EXPORT gboolean fl_binary_messenger_send_response(
     FlBinaryMessenger* self,
-    FlBinaryMessengerResponseHandle* response_handle,
+    FlBinaryMessengerResponseHandle* response_handle_,
```

Comment:
How strong is this convention in gstyle? Trailing underscore means ivar in Google C++ style, so I find this really confusing; if we could do `response_handle` and `owned_response_handle` or something instead that would be better.

Although from a higher level: If this is passing ownership in, can we use a `g_autoptr` in the API instead, so that it's unambiguous and foolproof?

Reply:
I don't know of a standard style, happy to change to `owned_response_handle` to make this clearer.

Sadly this is the best method I know of to do this. Logically the g_autoptr would be in the parameter definition but that's not possible in C:
```
../../flutter/shell/platform/linux/fl_binary_messenger.cc:145:5: error: 'cleanup' attribute only applies to local variables [-Werror,-Wignored-attributes]
    g_autoptr(FlBinaryMessengerResponseHandle) response_handle,
```

This made me notice that the `(transfer full)` notation was missing from these functions. This shows the API user in the documentation the ownership rules and allows bindings to handle this automatically.